modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -367,7 +367,7 @@ const EditorManager: EditorManager = {
     };
 
     const findTargets = (options: RawEditorOptions): HTMLElement[] => {
-      if (Env.browser.isIE() && Env.browser.version.major < 11) {
+      if (Env.browser.isIE() || Env.browser.isEdge()) {
```

Comment:
we do support Chrome Edge, check if this isEdge check knows the difference between IE edge and Chrome edge

Reply:
This is only legacy edge. Chromium Edge and other chromium variants fall under the `isChrome` check.